lib/spack/spack/binary_distribution.py

```diff
@@ -1356,7 +1356,7 @@ def push(specs, push_url, specs_kwargs=None, **kwargs):
         try:
             _build_tarball(node, push_url, **kwargs)
         except NoOverwriteException as e:
-            warnings.warn(str(e))
```

Comment:
Is it this change to reduce the number of calls? I'd be curious to know why in that case.

Reply:
I just thought we did not really use warnings